### PR TITLE
Add Rustface to the feature detection docs

### PR DESCRIPTION
**Pull request by nealtodd**

### CODE REVIEW

File: docs/advanced_topics/images/feature_detection.rst [hunk 1]
```diff
@@ -5,17 +5,22 @@ Feature Detection
 
 Wagtail has the ability to automatically detect faces and features inside your images and crop the images to those features.
 
-Feature detection uses OpenCV to detect faces/features in an image when the image is uploaded. The detected features stored internally as a focal point in the ``focal_point_{x, y, width, height}`` fields on the ``Image`` model. These fields are used by the ``fill`` image filter when an image is rendered in a template to crop the image.
+Feature detection uses third-party tools to detect faces/features in an image when the image is uploaded. The detected features stored internally as a focal point in the ``focal_point_{x, y, width, height}`` fields on the ``Image`` model. These fields are used by the ``fill`` image filter when an image is rendered in a template to crop the image.
```
Comment (tomdyson) [inline docs/advanced_topics/images/feature_detection.rst]: The detected features **are** stored internally

File: docs/advanced_topics/images/feature_detection.rst [hunk 2]
```diff
@@ -58,8 +63,53 @@ You can test that OpenCV can be seen by Wagtail by opening up a python shell (wi
 If you don't see an ``ImportError``, it worked. (If you see the error ``libdc1394 error: Failed to initialize libdc1394``, this is harmless and can be ignored.)
 
 
+Rustface
+~~~~~~~~
+
+Rustface_ is python library with prebuilt wheel files provided for Linux and macOS. Although implemented in Rust it is pip-installable:
```
Comment (tomdyson) [inline docs/advanced_topics/images/feature_detection.rst]: is python library -> is a Python library

File: docs/advanced_topics/images/feature_detection.rst (hunk 1 above)
Comment (nealtodd) [reply docs/advanced_topics/images/feature_detection.rst]: Fixed by 5ea9e81

File: docs/advanced_topics/images/feature_detection.rst (hunk 2 above)
Comment (nealtodd) [reply docs/advanced_topics/images/feature_detection.rst]: Fixed by 5ea9e81

File: docs/advanced_topics/images/feature_detection.rst [hunk 3]
```diff
@@ -58,10 +63,55 @@ You can test that OpenCV can be seen by Wagtail by opening up a python shell (wi
...
+This should be done somewhere that gets run on application startup:
+
+ .. code-block:: python
+
+    from willow.registry import registry
+    import rustface.willow
+
+    registry.register_plugin(rustface.willow)
+
+For example, in an app's AppConfig.onready_.
```
Comment (kaedroho) [inline docs/advanced_topics/images/feature_detection.rst]: I think this method is just `AppConfig.ready`

Comment (nealtodd) [reply docs/advanced_topics/images/feature_detection.rst]: It is indeed even though the link was correct. Fixed by ea95b99. Thanks for the spot.

**Comment by nealtodd** [comment]
Hi @kaedroho - can we still get this merged in?, after fixing the subsequent conflicts from a later merged PR.

**Comment by lb-** [comment]
Merged in via 2b35a852fed7dfa998e87b97721f1e35062ecf65 
Thanks heaps @nealtodd

**Comment by kaedroho** [comment]
@nealtodd Thanks again and sorry for the delay. Not sure how I missed your last comment

**Pull request closed without merging**